src/main/resources/views/CommercialVehicles/failWelsh.hbs

```diff
@@ -120,7 +120,7 @@
                               </span>
                               <span class="heading-info__title-right">
                               <span class="heading-info__title-message">
-                                  {{regOrIdHeading}}
+                                  {{regOrIdHeadingWelsh}}
```

Comment:
Can you look at line 36 please

```
{{presentedDocumentNameFail}}/{{versionNumberFail}}
```

Whilst the version numbers are currently the same, there may come a point in the future where VTP30 and VTP30W versions differ, so this should be pointing to {{versionNumberFailWelsh}}